From ca79b88a0d7b202bd8781108046e20dffb5ae1c1 Mon Sep 17 00:00:00 2001
From: kboeluek <kboeluek@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 3 Apr 2014 18:07:20 +0000
Subject: [PATCH 152/462] MANTIS 0031232 change CUSS*.jar: add version
 information to MANIFEST.MF

- added additional information to the MANIFEST.MF of jars created by XMLBeans
- updated ANT to the current version.

git-svn-id: svn://localhost/SelfServiceBuild/trunk@222 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntTargets.xml | 3 ++-
 SelfServiceBuild/Build/scripts/ant.bat     | 4 ++--
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntTargets.xml b/SelfServiceBuild/Build/java/AntTargets.xml
index f6ea437..759efbe 100644
--- a/SelfServiceBuild/Build/java/AntTargets.xml
+++ b/SelfServiceBuild/Build/java/AntTargets.xml
@@ -88,11 +88,12 @@
     <!-- Die angegebene Schema-Datei wird compiliert. -->
     <echo message="Compiling XML-Beans Schema: ${SchemaFile}"/>
     <xmlbean schema="${SchemaFile}" srcgendir="${GenXMLSrcDir}/${GenFile}" destfile="${JarDir}/${GenFile}.jar" classpath="${JarXMLBeans};${JarJSR173}"/>
+    <jar destfile="${JarDir}/${GenFile}.jar" manifest="${ManifestDir}/${ManifestName}" update="true"/>
 </target>
 
 <!-- create-xml-bean ================================================= -->
 
-<target name="create-xml-bean" if="project.xmlBean">
+<target name="create-xml-bean" if="project.xmlBean" depends="create-manifest">
     <mkdir dir="${GenSrcDir}"/>
 
     <!-- Alle Schema-Dateien werden compiliert. -->
diff --git a/SelfServiceBuild/Build/scripts/ant.bat b/SelfServiceBuild/Build/scripts/ant.bat
index c129329..2b8d8f9 100644
--- a/SelfServiceBuild/Build/scripts/ant.bat
+++ b/SelfServiceBuild/Build/scripts/ant.bat
@@ -5,7 +5,7 @@ rem # /vob/SelfServiceBuild/Build/scripts/ant.bat
 rem #
 rem ######################################################################
 
-set VERSION_ANT=1.7.0
+set VERSION_ANT=1.9.3
 set DIR_ANT=%DIR_GLOBALS%\Tools\ant\%VERSION_ANT%\apache-ant-%VERSION_ANT%
 
-%DIR_ANT%\bin\ant.bat %1 %2 %3 %4 %5 %6 %7 %8 %9
+%DIR_ANT%\bin\ant.bat %1 %2 %3 %4 %5 %6 %7 %8 %9
\ No newline at end of file
-- 
2.41.0.windows.1

